src/manual/using/actions/spec.md

```diff
@@ -281,6 +281,39 @@ Note that the result of rendering `hookPayload` becomes the context for another
 JSON-e rendering performed by the hooks service.  Do not confuse the two
 operations!
 
+### Choosing a Kind
+
+The "task" kind requires that the user invoking the task has all of the scopes
+necessary to run the action task (and to create any further tasks). Such scopes
+might allow a user to perform undesirable actions with the Taskcluster API.
+
+The "hook" kind allows more control over users' permissions. The key
+observation is that `hooks.triggerHook` is governed by a scope of the form
+`hooks:trigger-hook:<hookGroupId>/<hookId>`, but the task it creates uses role
+`hook-id:<hookGroupId>/<hookId>`. The method also verifies its payload against
+the hook's `triggerSchema`.  This allows a controlled privilege escalation:
+only clients with the appropriate `trigger-hook` scope may trigger a hook, and
+only if they provide a payload matching the schema. If both of those conditions
+are met, then the resulting task may execute with permissions the originating
+client does not posses.
+
+To design a hook-based action, start with the hook and consider the minimal
```

Comment:
Could this be prefixed `**Example, **` (in markdown)

That way it's easy to see that I don't care :)

Reply:
It's not an example, though..

Reply:
Could it have an informal title or something...